### DISCUSSION TITLE
Custom Manager Support using Regex Help with ARG

### DISCUSSION THREAD

**Discussion by lucdiniz** [renovatebot/renovate#19398] [Request Help] [answered]
Hello,

Anyone expert on creating regex for docker, with Custom Manager Support using Regex, can help?

````
ARG PARENT_IMAGE=python
ARG PARENT_VERSION=3.9.0-alpine

FROM ${PARENT_IMAGE}:${PARENT_VERSION}
````

Thanks in advance

**Comment by Churro** [comment, marked as answer]
A very good explanation on how to achieve this can be found here: https://docs.renovatebot.com/modules/manager/regex/

In any case, for the Dockerfile snippet you posted, a custom regex pattern isn't even necessary. Current versions of renovate can detect and resolve this automatically.

**Reply by lucdiniz** [reply]
Thanks @Churro I was using an old version 23.73.2

**Reply by viceice** [reply]
😱 That's pretty old and unsupported. we're current at v34.x. So you should update renovate. Otherwise most of the docs won't work on your old version.
